rotkehlchen/tests/db/test_reports.py

```diff
@@ -1,10 +1,26 @@
-from rotkehlchen.accounting.pnl import PnlTotals
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.pnl import PNL, PnlTotals
+from rotkehlchen.accounting.structures.processed_event import ProcessedAccountingEvent
+from rotkehlchen.constants import ONE, ZERO
+from rotkehlchen.constants.assets import A_DAI, A_ETH
+from rotkehlchen.db.filtering import ReportDataFilterQuery
 from rotkehlchen.db.reports import DBAccountingReports
 from rotkehlchen.db.settings import DBSettings
+from rotkehlchen.fval import FVal
 from rotkehlchen.tests.utils.constants import A_GBP
+from rotkehlchen.types import Location, Price, Timestamp
+from rotkehlchen.utils.misc import create_order_by_rules_list, timestamp_to_date
 
 
-def test_report_settings(database):
+def convert_timestamp(timestamp: Timestamp):
+    """Converts the timestamp to string"""
+
+    return timestamp_to_date(timestamp)
+
+
+def setup_db_account_settings(database):
+    """Setup accounting reports and settings db"""
+
     dbreport = DBAccountingReports(database)
```

Comment:
no need from empty lines here

```suggestion
    """Setup accounting reports and settings db"""
    dbreport = DBAccountingReports(database)
```

same between dbreport and settings